### ISSUE TITLE
Gallery fails to download - Parser failed to obtain required properties from gallery page

### ISSUE THREAD

**Issue by herkalurk** [patrickkfkan/imagefap-dl#1]
Attempting to download a full gallery, I thought the problem might be I can't see a user who uploaded the gallery, so I used about every option I could find on the readme to not capture that data, simply just download the images.  Still getting an error saying failed to obtain required properties.  If I only want the images,unsure what other properties I can't get.  Debugging logging didn't show any extra data.

I was able to previously download a different gallery, but that gallery did have a clear link to the user who uploaded it, where as this one didn't.

```
imagefap-dl -j -m --log-level debug -d - --out-dir gifs https://www.imagefap.com/pictures/8037543/

imagefap-dl v1.1.0 ImageFap gallery downloader

Log level: debug
Created ImageFapDownloader instance with config:  {
  outDir: '/media/raid5/cuntkittn/gifs',
  dirStructure: { uploader: false, folder: false, gallery: false },
  request: {
    maxRetries: 3,
    maxConcurrent: 10,
    minTime: { page: 2000, image: 200 }
  },
  overwrite: false,
  saveJSON: false,
  saveHTML: false,
  targetURL: 'https://www.imagefap.com/pictures/8037543/'
}

Proceed (Y/n)? y
Feb 18 16:58:13: info: ImageFapDownloader: Fetching gallery contents from "https://www.imagefap.com/pictures/8037543/"
Feb 18 16:58:13: debug: ImageFapDownloader: Fetch page "https://www.imagefap.com/pictures/8037543/"
Feb 18 16:58:14: error: ImageFapDownloader: Error fetching gallery "https://www.imagefap.com/pictures/8037543/" - download skipped:  Parser failed to obtain required properties from gallery page
Feb 18 16:58:14: info: ImageFapDownloader: Download complete
Feb 18 16:58:14: info: ImageFapDownloader: --------------
Feb 18 16:58:14: info: ImageFapDownloader: Download stats
Feb 18 16:58:14: info: ImageFapDownloader: --------------
Feb 18 16:58:14: info: ImageFapDownloader: Processed galleries: 0
Feb 18 16:58:14: info: ImageFapDownloader: Downloaded images: 0
Feb 18 16:58:14: info: ImageFapDownloader: Skipped existing images: 0
Feb 18 16:58:14: info: ImageFapDownloader: Errors: 0
```

**Comment by herkalurk** [comment]
Based on what I'm seeing in the code, you expect to have the uploader in the data.  IMO it should be optional if I'm not going to create a folder structure based on the uploader and simply want to grab the images.

https://github.com/patrickkfkan/imagefap-dl/blob/master/src/lib/parsers/Parser.ts#L215

**Comment by patrickkfkan** [comment]
You are correct. I'll add support for anonymous users.

**Comment by patrickkfkan** [comment]
Resolved by 26eecf3

**Comment by patrickkfkan** [comment]
v1.1.1 published to NPM
